Lib/vanilla/vanillaBase.py

```diff
@@ -543,9 +543,8 @@ def _breakCycles(view):
     """
     Break cyclic references by deleting _target attributes.
     """
-    if hasattr(view, "vanillaWrapper"):
+    if view.respondsToSelector_("vanillaWrapper"):
         obj = view.vanillaWrapper()
-        if hasattr(obj, "_breakCycles"):
-            obj._breakCycles()
+        obj._breakCycles()
```

Comment:
the vanillaWrapper object could be None, fe when weakref has no reference of that object anymore

testing `if obj is not None:` will not slow down breakCycles